README.md

```diff
@@ -43,7 +44,8 @@ Our team will be creating a website to help people find recipes for maintaining
 - Users will be able to set their own preferences for food recipes; such as the level of spiciness, the food cultures, and many more.
 - The system will save the information and organize the pages to fit the user preference.
 
-4. Users will be able to filter food recipes by:
+4. Filter
+- Users will be able to filter food recipes by:
```

Comment:
For each filter type, we should give an explanation. For example, the cooking difficulty has easy, medium and hard, and we can add why we need this feature. Two bullet points for each one.

Reply:
Ok. I will review and send another pull request for review.